docs/docs/features/combos.md

```diff
@@ -29,6 +30,7 @@ Combos configured in your `.keymap` file, but are separate from the `keymap` nod
 - `key-positions` is an array of key positions. See the info section below about how to figure out the positions on your board.
 - `bindings` is the behavior that is activated when the behavior is pressed.
 - (advanced) you can specify `slow-release` if you want the combo binding to be released when all key-positions are released. The default is to release the combo as soon as any of the keys in the combo is released.
+- (advanced) you can specity a list of layers (`layers = <0 1...>`) to limit a combo to specific layers. this is an _optional_ parameter and defaults to `-1` which is global scope.
```

Comment:
I think this should not be in the 'advanced' section but in the default.

Reply:
Move up in docs to be the last item before advanced and removed advanced modifier from text